backend/backend/settings/deps/ledger.py

```diff
@@ -142,7 +142,9 @@ def update_client_with_discovery(client, discovery_results):
                     'url': url,
                     'grpcOptions': {
                         'grpc-max-send-message-length': 15,
-                        'grpc.ssl_target_name_override': peer_info['endpoint'].split(':')[0]
+                        'grpc.ssl_target_name_override': peer_info['endpoint'].split(':')[0],
+                        'grpc.max_send_message_length': -1,
+                        'grpc.max_receive_message_length': -1
```

Comment:
Is it intended to have the keys `grpc-max-send-message-length` and  `grpc.max_send_message_length` in `grpcOptions`? 

It looks a bit confusing: it's the same name with different naming styles and different values (15 vs -1).

Reply:
Yes nice catch, I would like to have a setup to test it but I need @thibaultrobert  to do that easily I think